corehq/apps/hqwebapp/tasks.py

```diff
@@ -0,0 +1,57 @@
+from celery.task import task
+from celery.utils.log import get_task_logger
+from django.conf import settings
+from django.core.mail import send_mail
+from dimagi.utils.django.email import send_HTML_email
+
+logger = get_task_logger(__name__)
+
+
+@task(bind=True, default_retry_delay=10 * 60, max_retries=10)
```

Comment:
I also mentioned on the ticket, but we might want to consider more carefully what queue we use for this. Right now it looks like it will use the `main` queue, which is probably ok, but does mean that if it gets really backed up with exports we will see delays in sending emails (which could be frustrating for people during the sign up process)